### DISCUSSION TITLE
KasmVNC for dell wyse thinos?

### DISCUSSION THREAD

**Discussion by unecc** [kasmtech/KasmVNC#237] [General]
Hi i use a little dell 3040 for connect in my windows virtual machine. It use RPD protocol that's inside the 3040 os

Can i use KasmVNC for connect in my linux virtual machine of proxmox?
Thanks

**Comment by mmcclaskey** [comment]
I did not fully understand your setup and question. However... If you are running a supported Linux OS on a virtual machine inside of Proxmox, you can connect to that Linux VM using any system with a browser. Since KasmVNC is web native, there is nothing to install on the client, only on the server side.
